tests/scripts/multus/host-cfg-ds.yaml

```diff
@@ -0,0 +1,68 @@
+####################################################################################################
+# this daemonset configures each minikube or KinD node to route to the multus public net
+# it is not suitable for production
+####################################################################################################
+apiVersion: apps/v1
```

Comment:
This daemonset is only used for the integration tests, right? I don't see it mentioned in the docs.

Reply:
Right. It is test only